test/fixtures/spread-attributes/expected.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+elementOpenStart("div", "test", ["class", "test", "key", "test"]);
```

Comment:
For support, I think so. Idom will [initialize stored data](https://github.com/google/incremental-dom/blob/master/src/node_data.js#L105-L107) (if it doesn't exist) with a key from the `key` attribute.